integrationExamples/videoModule/index.html

```diff
@@ -0,0 +1,113 @@
+<!--
+
+-->
+
+<html>
+
+<head>
+    <script async src="../../build/dev/prebid.js"></script>
+    <script>
+      var adUnits = [{
+        code: 'div-gpt-ad-51545-0',
+        sizes: [[600, 500]],
+        mediaTypes: {
+          video: {
+            sizes: [[600, 500]]
+          }
+        },
+        video: {
+          divId: 'vid-div-1',
+          adServer: {
+            vendorCode: "gam",
+            params: {
+              iu: '/19968336/prebid_cache_video_adunit',
+              cust_params: {
+                section: 'blog',
+                anotherKey: 'anotherValue'
+              },
+              output: 'vast'
+            }
+          },
+        },
+
+        // Replace this object to test a new Adapter!
+        bids: [{
+          bidder: 'ix',
+          params: {
+            siteId: '300',
+            size: [600, 500]
+          }
+        }]
+      }];
+
+      var pbjs = pbjs || {};
+      pbjs.que = pbjs.que || [];
+    </script>
+    <script>
+      pbjs.que.push(function () {
+        pbjs.onEvent('adImpression', event => {
+          console.log('adImpression: ', event);
+        });
+        pbjs.addAdUnits(adUnits);
+        pbjs.setConfig({
+          video: {
+            providers: [{
+              divId: 'vid-div-1',
+              vendorCode: 1,
+              playerConfig: {
+                licenseKey: "INSERT LICENSE KEY HERE",
```

Comment:
Some players require providing a license key. The user does not need to publish his key in the PR. He would provide it in his local build for testing. We will be adding a license key to this PR for demo purposes. 
I a publisher's video player does not require a license key to be passed in, they can leave this blank. Alternatively, they can use their player's API to provide the license key; it is not required to pass it in via the video module's config; the video module's config is just for convenience.

Reply:
anything that the end user needs to pass to `setConfig` is available to anyone who knows how to right click and inspect source. If these keys are something they pay for, it seems like something we should at least make clear and/or discourage.

Reply:
The requirement of using a key is not up to us. If the player requires one, there's nothing we can do about it. This is just a convenience for the publisher to configure the player in one place. 
I agree with adding a warning in the docs and to let the reader know that if a safer alternative is available, that they should go with the latter.

Reply:
I know that many companies will automatically terminate service to api keys that have been leaked on the internet, so I'm not so sure that this would necessarily be a convenience - if it's a secret they need to protect, we should not allow them to make it public so easily. But again, I don't know if that's the case for these particular keys.